include/seqan3/alphabet/quality/concept.hpp

```diff
@@ -47,13 +50,20 @@ namespace seqan3
 namespace detail
 {
 
-/*!
- * Internal conversion functions of a quality alphabet.
- * A quality has three representations:
- *    a) internal unsigned integer, simply called "value"
- *    b) an integer representing the phred score, calibration is machine-dependent
- *    c) a single-letter encoding corresponding to a phred score
+/*!\interface seqan3::internal_quality_concept <>
+ * \extends seqan3::alphabet_concept
+ * \brief A concept that indicates whether an alphabet represents quality scores.
+ * \ingroup quality
+ *
+ * In addition to the requirements for seqan3::alphabet_concept, the
+ * internal_quality_concept introduces a requirement for conversion functions
+ * from and to a Phred score.
+ *
+ * \par Concepts and doxygen
+ * The requirements for this concept are given as related functions and metafunctions.
+ * Types that satisfy this concept are shown as "implementing this interface".
```

Comment:
as documented in the last issues and PRs, please remove the internal_quality_concept